addon/mixins/validatable-input.js

```diff
@@ -165,7 +165,7 @@ export default Ember.Mixin.create({
     } else {
       parent.addClass('has-error');
       element.next('.input-error').remove();
-      element.after('<p class="input-error" role="alert">' + errorMessage + '</p>');
+      element.after(`<label class="input-error" role="alert" for="${element.attr("id")}">${errorMessage}</label>`);
```

Comment:
I'm not entirely sure about the label tag here. Isn't this label only to be used on par with an input? That's not really the case here. For the ${} syntax, what's the browser support of it (I think this is ES6 syntax and is converted to ES5 in Ember-CLI right?)